rkt/run.go

```diff
@@ -86,6 +86,8 @@ func init() {
 	cmdRun.Flags().Var((*appAsc)(&rktApps), "signature", "local signature file to use in validating the preceding image")
 	cmdRun.Flags().Var((*appExec)(&rktApps), "exec", "override the exec command for the preceding image")
 	cmdRun.Flags().Var((*appMount)(&rktApps), "mount", "mount point binding a volume to a path within an app")
+	cmdRun.Flags().Var((*appMemoryLimit)(&rktApps), "memory", "memory limit")
```

Comment:
I think we should give more information here:

* it's for the preceding app
* syntax